frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/DataFormat/index.tsx

```diff
@@ -1,15 +1,11 @@
-import {
-  Intent,
-  PopoverInteractionKind,
-  Position,
-  Radio,
-  RadioGroup,
-  Tooltip,
-} from "@blueprintjs/core";
-import * as React from "react";
-import { FC } from "react";
+import React, { FC } from "react";
 import { DATASET_ASSET_FORMAT } from "src/common/entities";
-import { Section, Title } from "../common/style";
+import { FormControl, FormLabel, RadioGroup } from "@mui/material";
+import { InputRadio, Tooltip } from "@czi-sds/components";
```

Comment:
`Tooltip` is now from the `czi-sds` library - and to simplify the code we wrap the radio button with a `span` to allow the tooltip to work, should the radio button be disabled as per the [warning provided by the library](https://github.com/chanzuckerberg/sci-components/blob/main/packages/components/src/core/Tooltip/index.tsx#L28).

Reply:
I wonder if we should add a comment in the code where `<span>` is used, so people remember about this gotcha!